app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,47 @@
+class IngredientsController < ApplicationController
+  before_action :set_ingredient, only: %i[ show edit update destroy ]
+
+  def index
+    @ingredients = Ingredient.order(:id)
+  end
+
+  def new
+    @ingredient = Ingredient.new
+  end
+
+  def create
+    @ingredient = Ingredient.new(ingredient_params)
+
+    if @ingredient.save
+      redirect_to ingredients_url, notice: t('.success')
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @ingredient.update(ingredient_params)
+      if params[:ingredient][:destroy_on_save] == '1'
+        @ingredient.image.purge
+      end
```

Comment:
We can use single line if here.
Aren't you following rubocop rules?
First, take out a branch from main and fix all rubocops.